stlTranslator/SourceFiles/read.cpp

```diff
@@ -10,23 +11,22 @@ Read::Read()
 }
 
 void Read::read() {
-    ifstream myFile("sphere.stl");
-
-    if (myFile.is_open()) {
-        while (getline(myFile, line)) {
-            if (line.find("vertex") != string::npos) {
-                int pos = line.find("vertex");
-                data += line.substr(pos + 7);
-                data += ' ';
```

Comment:
can be writtern as 
"data += line.substr(pos + 7) + '  ' ; "

Reply:
The entire read.cpp file is changed